app/controllers/account/users_controller.rb

```diff
@@ -1,14 +1,15 @@
 class Account::UsersController < ApplicationController
   before_action :authenticate_user!
-  before_action :authorize_admin!
+  before_action :authorize_admins!
 
   def index
-    @search = current_tenant.users.ransack(params[:q])
+    users = current_user.super_admin? ? super_admin_collection : admin_collection
```

Comment:
create method collection
and return there:
current_user.super_admin? ? super_admin_collection : admin_collection